src/Microsoft.IdentityModel.Tokens/Validation/ValidationParameters.cs

```diff
@@ -112,14 +118,25 @@ public ValidationParameters()
         public AlgorithmValidator AlgorithmValidator { get; set; }
 
         /// <summary>
-        /// Gets or sets a delegate that will be used to validate the audience.
+        /// Allows overriding the delegate that will be used to validate the audience.
         /// </summary>
         /// <remarks>
-        /// If set, this delegate will be called to validate the 'audience', instead of default processing.
+        /// If set, this delegate will be responsible for validating the 'audience', instead of default processing.
```

Comment:
We need to update the wording around `if set`, since it will always be set at least to our default.
I am okay doing this after this PR, though, as I need this merged for JWS/JWE validation.

Reply:
Will update on the next pass for Audience validator then.